primer_designer/views.py

```diff
@@ -170,7 +164,15 @@ def create(request, regions_form):
 
     # unique name of date and random 5 char str
     outname = f'{time_stamp()}-{random_string()}'
-    out_dir = f'{Path(__file__).parent.parent.absolute()}/static/tmp/{outname}/'
+
+    # making an output directory in /primer_designer for output PDF
+    parent_path = Path(__file__).parent.parent.absolute()
+    Path(f'{parent_path}/primer_designer/output/').mkdir(
+        parents=True, exist_ok=True)
+    primer_output_path = f'{parent_path}/primer_designer/output/'
+
+    Path(f'{parent_path}/static/tmp').mkdir(parents=True, exist_ok=True)
+    out_dir = f'{parent_path}/static/tmp/{outname}/'
```

Comment:
still need to think of a nice way to keep this cleared, this exists which might be a nice option: https://pypi.org/project/django-crontab/

Reply:
yup, next PR on Dockerfile and scheduled task